File: cfgs/java_cfgs_foo/FIND_UNIT_DIGIT_X_RAISED_POWER_Y.java
Nodes
1;special;;;
2;statement;int res = 1;14;
3;statement;int i = 0;15;
4;conditional;foo_1 < foo_2;16;
5;statement;res = (foo_1 * foo_2) % 10;17;
6;statement;i++;17;
7;statement;return foo_1;18;
8;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;7;
4;6;
5;4;
6;8;
7;

File: cfgs/pythoncfgs/FIND_UNIT_DIGIT_X_RAISED_POWER_Y.py
Nodes
1;statement;foo_1 = 1;8;
2;conditional;fun_1(foo_1);9;
3;statement;return foo_1;11;
4;statement;foo_1 = foo_2 * foo_3 % 10;10;
5;exit;;
Edges
0;2;
1;3;4;
2;5;
3;2;
4;